packages/salesforcedx-apex-replay-debugger/test/unit/states/userDebugState.test.ts

```diff
@@ -63,7 +63,8 @@ describe('User debug event', () => {
   it('Should link to anonymous specific frame', () => {
     const frame = {
       name: EXEC_ANON_SIGNATURE,
-      source: { name: 'foo.log' }
+      // eslint-disable-next-line no-global-assign
```

Comment:
Why is this eslint directive necessary here and not on line 91?

Reply:
good catch. I forgot to remove it